### ISSUE TITLE
Incorrect daydatetime format string

### ISSUE THREAD

**Issue by sahutd** [uniplaces/carbon#68]
```go
func TestTimestampFormat(t *testing.T)  {
	c, err := carbon.CreateFromTimestamp(1624647416, "UTC")
	assert.Nil(t, err)
	assert.Equal(t, "Fri, Jun 25, 2021 6:56 PM", c.DayDateTimeString())
}
```
The above timestamp is from june 25 6:56pm utc
![image](https://user-images.githubusercontent.com/5676959/123473656-7c6bdb80-d616-11eb-9464-fcca3405baa5.png)

```
Error:      	Not equal: 
        	            	expected: "Fri, Jun 25, 2021 6:56 PM"
        	            	actual  : "Fri, Aug 25, 2021 6:56 PM"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-Fri, Jun 25, 2021 6:56 PM
        	            	+Fri, Aug 25, 2021 6:56 PM
        	Test:       	TestTimestampFormat
```

Issue is due to wrong format string used in 
```
	DayDateTimeFormat   = "Mon, Aug 2, 2006 3:04 PM"

```

**Comment by sahutd** [comment]
Do let me know if the project is accepting PRs. i can fix the above issue.

**Comment by gouguoyin** [comment]
you can try [golang-module/carbon](https://github.com/golang-module/carbon)，a simple, semantic and developer-friendly golang package for datetime.Its result is `Fri, Jun 25, 2021 6:56 PM`
![image](https://user-images.githubusercontent.com/13517412/128464756-01734ad8-7e61-4daf-aba2-057a7d1579e8.png)

**Comment by Rossalli** [comment]
Hi @rntdrts I did a fix for this issue on PR #71
